deployment/common/changeset/internal/link_token.go

```diff
@@ -0,0 +1,36 @@
+package internal
+
+import (
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/types"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/shared/generated/link_token"
+)
+
+// DeployLinkTokenContract deploys a link token contract to the provided chain.
+func DeployLinkTokenContract(
```

Comment:
nit could probably just inline this actually in the changeset, I know we do this elsewhere but we're thinking its overkill and intend to inline the deploy.go functions inside the changesets.

Reply:
ah nice, i had it right in my first iteration but then i followed the MCMS and moved the logic in to `internal`, let me bring the original back as i prefer that too!